### PR TITLE
Refactor the exponent operation

### PR DESCRIPTION
**Pull request by mwinckler**
This replaces the inline power operation with a function.

### CODE REVIEW

File: calculator.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@ def divide(operand_1, operand_2)
   return operand_1 / operand_2
 end
 
+# Raise operand_1 to the operand_2th power
```
Comment (mwinckler) [inline calculator.rb]: This comment explains _what_, not _why._ It's not a very good comment. I think we should remove it.

Comment (ShortPeople) [inline calculator.rb:17]: This is the bad comment?

Comment (The-Real-Puddleglum) [inline calculator.rb:17]: replace 2th with 2nd.

Comment (ShortPeople) [reply calculator.rb:17]: ShortPeople concur with comment removal.

File: calculator.rb [hunk 2]
```diff
@@ -14,6 +14,11 @@ def divide(operand_1, operand_2)
...
+def power(operand_1, operand_2)
+  return operand_1 ** operand_2
```
Comment (The-Real-Puddleglum) [inline calculator.rb:19]: Isn't the command `return` only used in classes to switch between different classes? So wouldn't this code take you to a (non-existent) class called "**operand_ ** operand_2**"?

**Comment by mwinckler** [comment]
Closing this example PR.

**Pull request closed without merging**